src/Mapping/Annotation/TrackingAwareAnnotationInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Doctrine Behavioral Extensions package.
+ * (c) Gediminas Morkevicius <gediminas.morkevicius@gmail.com> http://www.gediminasm.org
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Gedmo\Mapping\Annotation;
+
+/**
+ * @internal
+ */
+interface TrackingAwareAnnotationInterface extends Annotation
```

Comment:
instead of an interface extending Annotation, I would suggest putting those constants on an enum-like class (being final and forbidding instantiation thanks to a private constructor).
If the PHP requirements were allowing it, those events would be a good candidate for an actual enum, as it is a closed list.